video_emotion/extract_classifier.py

```diff
@@ -1,21 +1,26 @@
-from video_emotion.facial_recognition.facial_recognition import analyze_video
-from video_emotion.emotionTagger.face_emotion_extraction import classify_face
+import sys
 
+from facial_recognition.facial_recognition import analyze_video
+from emotionTagger.face_emotion_extraction import classify_face,classify_faces
 def classify_video(video_path,time_range=None):
     faces = analyze_video(video_path,time_range)
     angry_sum = disgust_sum = fear_sum = happy_sum = sad_sum = surprise_sum = neutral_sum = 0.0
     number_of_faces = 0
+    realFaces = []
     for key,value in faces.items():
         for face in value:
-            emotions = classify_face(face)
-            angry_sum += emotions[0]
-            disgust_sum += emotions[1]
-            fear_sum += emotions[2]
-            happy_sum += emotions[3]
-            sad_sum += emotions[4]
-            surprise_sum += emotions[5]
-            neutral_sum += emotions[6]
-            number_of_faces = number_of_faces + 1
+            realFaces.append(face)
+    faces = classify_faces(realFaces)
+
+    for emotions in faces:
+        angry_sum += emotions[0]
+        disgust_sum += emotions[1]
+        fear_sum += emotions[2]
+        happy_sum += emotions[3]
+        sad_sum += emotions[4]
+        surprise_sum += emotions[5]
+        neutral_sum += emotions[6]
+        number_of_faces = number_of_faces + 1
     return {
```

Comment:
I'd prefer having an empty line between the for loop and the return.

Reply:
Understandable, will do in the future but i don't think it's necessary

Reply:
Nvm it's in  my next branch